#The mode ID
string modeID
#Mode ID mapping
float64 ID
#Obstacle Type
string ObstacleType
#Obstacle Height
float64 height
#Obstacle wide
float64 wide
#Obstacle length
float64 length
#Obstacle number
uint32 number
#Obstacle Density
float64 density

